### ISSUE TITLE
In Google Chrome, NVDA does not read pop-up link information

### ISSUE THREAD

**Issue by RD-Brown** [nvaccess/nvda#9447]
When a pop-up link is used to provide additional contextual information, such as a word’s definition, NVDA does not read the additional information when the link is activated.  (TestNav, Pearson’s test delivery platform, is used to demonstrate this error inside Google Chrome.)

### Steps to reproduce:
1.	In Chrome, go to link: 
https://ref.testnav.com/client/index.html#login?username=LGN522573008&password=U637PDCA
2.	From the Test Start screen, press “B” until you reach the Start button, and press Enter.
3.	From the Section Start screen, press “B” until you reach the Start button, and press Enter.
4.	From the Question screen, press Tab until you reach the reading passage tabs, and Down Arrow to “Emancipation: A Life Fable.”
5.	Press Tab until you reach the first pop-up glossary word, “throve,” and press Enter.
6.	Press Down Arrow, and note that although “glossary definition” is announced, no definition is read.

### Actual behavior:
Definition is not voiced after link is activated and Down Arrow is pressed.

### Expected behavior:
Definition should be read.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
V2019.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Google Chrome V73.0.3683.86

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA?
Yes, V2018.4.1.

**Comment by ehollig** [comment]
I can reproduce this in Chrome and Firefox. The code snippet is:
``<span id="pluginde96d" class="glossary-term glossary-plugin" data-content="grew up well" data-title="Glossary Definition" data-placement="bottom" tabindex="0" data-hasqtip="0" aria-haspopup="true" role="link">throve</span><div class="visuallyhidden shadow-popup" aria-hidden="true" id="qtip-0-shadow-popup"><div class="shadow-popup-title">Glossary Definition</div><div class="shadow-popup-content">grew up well</div></div>``
@RD-Brown is it possible that these links only show the definition when focus is on the link, and when focus is moved away, the definition disappears?

**Comment by Adriani90** [comment]
I also don'T understand exactly the snippet, it seems the definition is visually:hidden and hast also aria-hidden=true. So actually the screen reader does not report it which is conform with aria spec. In order for this to work, the visualy:hidden atribute must be deleted.
cc: @MarcoZehe in case I am wrong here.

**Comment by Adriani90** [comment]
We didn't get any further updates from the initial author with regards to our concerns, I am closing this as abandoned though i believe NVDA behaves correctly according to aria spec in this case.
